cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -1653,12 +1655,16 @@ impl_runtime_apis! {
 			build_state::<RuntimeGenesisConfig>(config)
 		}
 
-		fn get_preset(id: &Option<sp_genesis_builder::PresetId>) -> Option<Vec<u8>> {
-			get_preset::<RuntimeGenesisConfig>(id, |_| None)
+		fn get_preset(id: &Option<PresetId>) -> Option<Vec<u8>> {
+			get_preset::<RuntimeGenesisConfig>(id, &genesis_config_presets::get_preset)
 		}
 
-		fn preset_names() -> Vec<sp_genesis_builder::PresetId> {
-			vec![]
+		fn preset_names() -> Vec<PresetId> {
+			vec![
+				PresetId::from("genesis"),
```

Comment:
The names are duplicated in `get_preset` function. I don't have a good idea how to solve this. We could use const as Olivier proposed in #2714.

Reply:
added `const`s in f3e7e4d